scripts/zones/Grauberg_[S]/npcs/34.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Grauberg [S]
+--  NPC:
+--  Quest - Seeing Blood-red
+-- pos -283 -55 -97
+-----------------------------------
+local ID = require("scripts/zones/Grauberg_[S]/IDs")
+require("scripts/globals/quests")
+require("scripts/globals/keyitems")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    if player:getCharVar("SeeingBloodRed") == 3 then
+        --player:startEvent(14)
```

Comment:
Once the BCNM is in, this should be uncommented so the BCNM-teleportation cutscene can be triggered.